### PR TITLE
add restaurant feature

### PR DESCRIPTION
**Pull request by kunmi02**
This project is to show my skills in javascript using webpack.

I set up and configured webpack.
I added images and also used object-oriented concept in the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+- HTML 
+- CSS
+- Linter
+- JS
+- VScode
+- Webpack
+
+## Demo Link
+ http://akinkunmi.me/restaurant-page/
```
Comment (descholar-ceo) [inline README.md:29]: [required] This live demo link doesn't reach to the real live demo of your restaurant page, it showing your readme file instead, could you fix that issue?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Description
+
+A simple JS based App for restaurant app.
+
+## Setup
+- Clone the project by running `git clone https://github.com/kunmi02/restaurant-page.git` on terminal or CMD
+- Run terminal or CMD from the project directory.
+- Run `npm install` if linters are required.
+- Run index.html in your Web Browser.
```
Comment (descholar-ceo) [inline README.md]: [required] This file `index.html` is not found on the root directory of your project, you should specify where to find it

File: README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
```
Comment (descholar-ceo) [inline README.md]: [required] All of the above steps needs clarification on the commands to use inorder to be able to performed any given action. You should not leave any room for assumption. Think of non-tech users who will need to test your app,

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @kunmi02  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implement all of the minimal requirements of this project, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [ ] To make your project more professional, you need to make sure that all of the info presented on the readme file are accurate, please read and fix all of inline comments I put on your readme file

# [Optional changes] :yellow_circle: 
- [ ] It will be very nice if you try to add some styling on your restaurant page
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Azeez Azeez** 
Nice job with the improvements for this project :muscle:  :muscle:
It meets most of the requirements :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:

- [ ] The package "webpack-dev-server" should be included in your package.json
- [ ] Necessary run commands should be in package.json, inside scripts:
`
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  }
`

- [ ] Add some styling for each tab


 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- [ ] I see that you are trying to work on tse's recommendation :muscle: 

To Improve :factory_worker: (REQUIRED)

- [ ] Some of your run commands don't perform as expected e.g. when I run the `npm start` command, I got some errors:


- [ ] All necessarily used packages should be included in package.json like `webpack`, `webpack-cli, webpack-dev-serve`r etc. Also, those packages **should be only added** as `devDependencies` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))
- [ ] Please, add styling to your webpage as it is part of project requirement.

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

**Review by oloomoses — APPROVED** [review]
# Congratulations, Project Approved. :muscle: :tada: :clap: 
**Great job working on the suggested changes from the previous review.
You have passed all the requirements for this project :muscle: 
Feel free to merge and close this pull request.**

**I wish you the best in your next project and see you at the top :rocket:** 
#### Happy Coding
_Slack: Oloo Moses_

**Pull request merged**